### DISCUSSION TITLE
I made this using nativewind

### DISCUSSION THREAD

**Discussion by crstnmac** [nativewind/nativewind#153] [Show and tell]
<img src="https://user-images.githubusercontent.com/20360870/183244987-f88b3c99-bf1e-4af1-8221-1cf90f58b277.png" alt="Screenshot_1659781068" width="300">

Thankyou for making this library ❤️🎉🚀

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
